README.md

```diff
@@ -155,6 +155,25 @@ e.g. disabling
 </init-param>
 ```
 
+##### Capture error messages
+
+It is possible to capture error messages that were saved using `HttpServletRequest.setAttribute`. The `error-message` init param must be configured to get the name of the attribute.
+
+e.g capture error messages from requests with attribute name equals to 'error-info'
+
+```xml
+<init-param>
+    <param-name>error-message</param-name>
+    <param-value>error-info</param-value>
+</init-param>
+```
+
+Your java code should look like this:
+
+```java
+req.setAttribute("error-info", "Page not found");
+```
+
```

Comment:
@werberson, is this standard?

Reply:
Yes, It is commonly used to share values in the request scope.

Reply:
Thank you @werberson!